scripts/gulp/gulpfile.ts

```diff
@@ -170,9 +145,6 @@ gulp.task('watchForE2E', gulp.series(
 *------------------------------------------------------------------------**/
 
 gulp.task('cyRunLaunchpadE2E', gulp.series(
```

Comment:
Is this no longer needed because the `build` has already been cached? This probably works as is in CI but ideally those gulp script that call the `build-prod` would have a depends on `build` in case someone tries to run this locally. Probably ok for now but we want to get to a place where local and CI run the same way